va-gov/pages/education/gi-bill/post-9-11/ch-33-benefit.md

```diff
@@ -1,20 +1,23 @@
 ---
-title: Check Benefit
-entryname: post-911-gib-status
-layout: page-react.html
-includeBreadcrumbs: false
-in_maintenance: false
-maintenance_line1: We’re sorry. Our system is temporarily down while we fix a few things. Please try again later.
+layout: page-react-sidebar.html
+title: Check Your Post-9/11 GI Bill Benefits
+display_title: Check Post-9/11 GI Bill Benefits
+description:
+concurrence:
 collection: education
 spoke: Manage Benefits
 order: 9
+entryname: post-911-gib-status
+in_maintenance: false
+maintenance_line1: We’re sorry. Our system is temporarily down while we fix a few things. Please try again later.
+breadcrumbs_override:
+  - path: education/
+    name: Education and Training
+  - path: education/about-gi-bill-benefits/
+    name: GI Bill
+  - path: education/about-gi-bill-benefits/post-9-11/
+    name: Post-9/11 GIBill
+  - path: education/gi-bill/post-9-11/ch-33-benefit/
+    name: Check Post-9/11 GI Bill Benefits
```

Comment:
I think it would be cool to see all pages with custom breadcrumbs written this way, so we only have to adapt breadcrumbs HTML in one location.